Description: Avoid error log noise under mod_perl
Author: Raphael Geissert <geissert@debian.org>
Reviewed-by: Axel Beckert <abe@debian.org>
Last-Modified: 2014-10-11
Bug-Debian: https://bugs.debian.org/680321

Index: libgeo-ip-perl/lib/Geo/IP.pm
===================================================================
--- libgeo-ip-perl.orig/lib/Geo/IP.pm	2014-10-11 03:47:46.259401287 +0200
+++ libgeo-ip-perl/lib/Geo/IP.pm	2014-10-11 03:53:21.359196991 +0200
@@ -43,40 +43,40 @@
 
 require Geo::IP::Record;
 
-sub GEOIP_STANDARD()     { 0; }    # PP
-sub GEOIP_MEMORY_CACHE() { 1; }    # PP
-sub GEOIP_CHECK_CACHE()  { 2; }
-sub GEOIP_INDEX_CACHE()  { 4; }
-sub GEOIP_MMAP_CACHE()   { 8; }    # PP
-sub GEOIP_SILENCE()      { 16; }
-
-sub GEOIP_UNKNOWN_SPEED()   { 0; }    #PP
-sub GEOIP_DIALUP_SPEED()    { 1; }    #PP
-sub GEOIP_CABLEDSL_SPEED()  { 2; }    #PP
-sub GEOIP_CORPORATE_SPEED() { 3; }    #PP
+sub GEOIP_STANDARD     { 0; }    # PP
+sub GEOIP_MEMORY_CACHE { 1; }    # PP
+sub GEOIP_CHECK_CACHE  { 2; }
+sub GEOIP_INDEX_CACHE  { 4; }
+sub GEOIP_MMAP_CACHE   { 8; }    # PP
+sub GEOIP_SILENCE      { 16; }
+
+sub GEOIP_UNKNOWN_SPEED   { 0; }    #PP
+sub GEOIP_DIALUP_SPEED    { 1; }    #PP
+sub GEOIP_CABLEDSL_SPEED  { 2; }    #PP
+sub GEOIP_CORPORATE_SPEED { 3; }    #PP
 
 BEGIN {
 
     #my $pp = !( defined &_XScompiled && &_XScompiled && !$TESTING_PERL_ONLY );
     my $pp = !defined &open;
 
-    sub GEOIP_COUNTRY_EDITION()        { 1; }
-    sub GEOIP_CITY_EDITION_REV1()      { 2; }
-    sub GEOIP_REGION_EDITION_REV1()    { 3; }
-    sub GEOIP_ISP_EDITION()            { 4; }
-    sub GEOIP_ORG_EDITION()            { 5; }
-    sub GEOIP_CITY_EDITION_REV0()      { 6; }
-    sub GEOIP_REGION_EDITION_REV0()    { 7; }
-    sub GEOIP_PROXY_EDITION()          { 8; }
-    sub GEOIP_ASNUM_EDITION()          { 9; }
-    sub GEOIP_NETSPEED_EDITION()       { 10; }
-    sub GEOIP_DOMAIN_EDITION()         { 11; }
-    sub GEOIP_COUNTRY_EDITION_V6()     { 12; }
-    sub GEOIP_ACCURACYRADIUS_EDITION() { 14; }
-    sub GEOIP_NETSPEED_EDITION_REV1()  { 32; }
+    sub GEOIP_COUNTRY_EDITION        { 1; }
+    sub GEOIP_CITY_EDITION_REV1      { 2; }
+    sub GEOIP_REGION_EDITION_REV1    { 3; }
+    sub GEOIP_ISP_EDITION            { 4; }
+    sub GEOIP_ORG_EDITION            { 5; }
+    sub GEOIP_CITY_EDITION_REV0      { 6; }
+    sub GEOIP_REGION_EDITION_REV0    { 7; }
+    sub GEOIP_PROXY_EDITION          { 8; }
+    sub GEOIP_ASNUM_EDITION          { 9; }
+    sub GEOIP_NETSPEED_EDITION       { 10; }
+    sub GEOIP_DOMAIN_EDITION         { 11; }
+    sub GEOIP_COUNTRY_EDITION_V6     { 12; }
+    sub GEOIP_ACCURACYRADIUS_EDITION { 14; }
+    sub GEOIP_NETSPEED_EDITION_REV1  { 32; }
 
-    sub GEOIP_CHARSET_ISO_8859_1() { 0; }
-    sub GEOIP_CHARSET_UTF8()       { 1; }
+    sub GEOIP_CHARSET_ISO_8859_1 { 0; }
+    sub GEOIP_CHARSET_UTF8       { 1; }
 
     #
     sub api {
